templates/inspec/tests/integration/configuration/mm-attributes.yml

```diff
@@ -315,4 +315,18 @@ node_template:
 node_group:
   name: inspec-node-group
   description: A description of the node group
-  size: 0
\ No newline at end of file
+  size: 0
+  
+spannerinstance:
+  config: regional-us-east1
+  name: spinstance
+  display_name: inspectest
+  node_count: 2
```

Comment:
Lets make `node_count` 1 so that this test isn't as expensive to run during integration testing

Reply:
done